### PR TITLE
Rename PARALLEL_TIMEOUT_IN_SECONDS to PARALLEL_JOB_TIMEOUT_IN_SECONDS

### PR DESCRIPTION
**Pull request by yguedidi**
A more precise name that express this configured timeout is per job

### CODE REVIEW

File: packages/Config/RectorConfig.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function parallel(int $seconds = 120, int $maxNumberOfProcess = 16, int $
         $parameters = $this->parameters();
         $parameters->set(Option::PARALLEL, true);
 
-        $parameters->set(Option::PARALLEL_TIMEOUT_IN_SECONDS, $seconds);
```
Comment (yguedidi) [inline packages/Config/RectorConfig.php:58]: @TomasVotruba @samsonasik is this considered a BC break?

File: src/Configuration/Option.php [hunk 2]
```diff
@@ -194,7 +194,7 @@ final class Option
      * @internal Use @see \Rector\Config\RectorConfig::parallel() instead with pass int $seconds parameter
      * @var string
      */
-    public const PARALLEL_TIMEOUT_IN_SECONDS = 'parallel-timeout-in-seconds';
+    public const PARALLEL_JOB_TIMEOUT_IN_SECONDS = 'parallel-job-timeout-in-seconds';
```
Comment (yguedidi) [inline src/Configuration/Option.php:197]: @TomasVotruba @samsonasik is this considered a BC break?

Comment (samsonasik) [reply src/Configuration/Option.php:197]: probably add comment is better than rename constant

Comment (yguedidi) [reply src/Configuration/Option.php:197]: I realize it's marked as internal, so shouldn't be an issue I think

Comment (TomasVotruba) [reply src/Configuration/Option.php:197]: This is only internal constant, so the change should be ok :+1:

**Pull request merged**